pkg/pillar/types/zedroutertypes.go

```diff
@@ -1870,6 +1871,38 @@ type NetworkXObjectConfig struct {
 type IpRange struct {
 	Start net.IP
 	End   net.IP
+	Size  int
+}
+
+// Contains used to evaluate whether an IP address
+// is within the range
+func (ipRange IpRange) Contains(ipAddr net.IP) bool {
+	start := ipRange.Start.To4()
+	end := ipRange.End.To4()
+	cur := ipAddr.To4()
```

Comment:
No, because no need to restrict this to IPv4. It is just two bytes.Compare calls and no need to call To4()